api/src/stampy_chat/env.py

```diff
@@ -1,41 +1,45 @@
 import os
 import openai
 import pinecone
-from discord_webhook import DiscordWebhook
 
 if os.path.exists('.env'):
     from dotenv import load_dotenv
     load_dotenv()
 else:
     print("'api/.env' not found. Rename the 'api/.env.example' file and fill in values.")
 
-
-OPENAI_API_KEY   = os.environ.get('OPENAI_API_KEY')
-PINECONE_API_KEY = os.environ.get('PINECONE_API_KEY')
-LOGGING_URL      = os.environ.get('LOGGING_URL')
-PINECONE_INDEX   = None
-
 FLASK_PORT = int(os.environ.get('FLASK_PORT', '3001'))
 
+LOG_LEVEL           = os.environ.get("LOG_LEVEL", "WARNING").upper()
+DISCORD_LOG_LEVEL   = os.environ.get("DISCORD_LOG_LEVEL", "WARNING").upper()
+DISCORD_LOGGING_URL = os.environ.get('LOGGING_URL')
+
+### OpenAI ###
+OPENAI_API_KEY   = os.environ.get('OPENAI_API_KEY')
 openai.api_key = OPENAI_API_KEY # non-optional
 
+### Pinecone ###
+PINECONE_API_KEY     = os.environ.get('PINECONE_API_KEY')
+PINECONE_ENVIRONMENT = os.environ.get('PINECONE_ENVIRONMENT', "us-east1-gcp")
+PINECONE_INDEX_NAME  = os.environ.get("PINECONE_INDEX_NAME", "alignment-search")
+PINECONE_INDEX       = None
+PINECONE_NAMESPACE   = os.environ.get("PINECONE_NAMESPACE", "alignment-search")  # "normal" or "finetuned" for the new index, "alignment-search" for the old one
```

Comment:
@tayler6000 once this gets merged, the following will have to be updated in the env:
```
PINECONE_API_KEY="5...<API key>...4"
PINECONE_INDEX_NAME="stampy-chat-ard"
PINECONE_ENVIRONMENT="us-central1-gcp"
PINECONE_NAMESPACE="normal"
```